superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -512,6 +495,28 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     );
   };
 
+  const {
+    ALERT_REPORTS_DEFAULT_WORKING_TIMEOUT,
+    ALERT_REPORTS_DEFAULT_CRON_VALUE,
+    ALERT_REPORTS_DEFAULT_RETENTION,
+  } = useSelector<any, AlertsReportsConfig>(state => state.common.conf);
+
+  const DEFAULT_ALERT = {
```

Comment:
nit: very minor detail, but as this is not technically a constant, it might be be worth considering renaming this to `defaultAlert`.

Reply:
Thanks for soon reviewing!
Renamed.